app/src/main/java/com/github/libretube/helpers/ImageHelper.kt

```diff
@@ -19,44 +19,53 @@ import com.github.libretube.util.DataSaverMode
 import java.nio.file.Path
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
+import java.io.File
 
 object ImageHelper {
     lateinit var imageLoader: ImageLoader
 
+    private val Context.coilFile get() = cacheDir.resolve("coil")
+
     /**
      * Initialize the image loader
      */
     fun initializeImageLoader(context: Context) {
-        val maxImageCacheSize = PreferenceHelper.getString(
-            PreferenceKeys.MAX_IMAGE_CACHE,
-            ""
-        )
+        val maxCacheSize = PreferenceHelper.getString(PreferenceKeys.MAX_IMAGE_CACHE, "128")
+        val allowsCache = maxCacheSize.isNotEmpty()
 
         imageLoader = ImageLoader.Builder(context)
             .callFactory(CronetHelper.callFactory)
+            .crossfade(true)
             .apply {
-                when (maxImageCacheSize) {
-                    "" -> {
-                        diskCachePolicy(CachePolicy.DISABLED)
-                    }
+                if (!allowsCache) {
```

Comment:
As we only use this condition once, I don't think it's reasonable to save `allowCache` in a variable.